verticapy/jupyter/extensions/chart_magic.py

```diff
@@ -192,30 +192,427 @@ def chart_magic(
     line: str, cell: Optional[str] = None, local_ns: Optional[dict] = None
 ) -> Union[Highstock, Highchart]:
     """
-    Draws  responsive charts using the High Chart  API:
-    https://api.highcharts.com/highcharts/
+    Draws responsive charts using the Matplotlib,
+    Plotly, or Highcharts library.
+    You can set your plotting library with the
+    set_option function.
     The returned object can be customized using the API
```

Comment:
as we have now the possibility to draw graphics using 3 different APIs. We need to indicate that the user should refer to the corresponding API (set_dict_options is specific to Highcharts.

Something like:
```
Different customization parameters are available for Plotly, Highcharts, and Matplotlib. 
    For a comprehensive list of customization features, please consult the documentation of the respective 
    libraries: `plotly <https://plotly.com/python-api-reference/>`_, `matplotlib <https://matplotlib.org/stable/api/matplotlib_configuration_api.html>`_ and `highcharts <https://api.highcharts.com/highcharts/>`_.
```
should work.